usaddress/__init__.py

```diff
@@ -61,72 +61,81 @@
                   'northeast', 'northwest', 'southeast', 'southwest'])
 
 STREET_NAMES = {
-    'allee', 'alley', 'ally', 'aly', 'anex', 'annex', 'annx', 'anx', 'arc',
```

Comment:
No problem! Alphabetical is definitely preferable here – the diff confusion is git's fault, not yours. Looks good to me.